### PR TITLE
Changed Supervisor related docs to recommend using separate config files

### PR DESCRIPTION
**Pull request by MarcinOrlowski**
Changed supervisor related docs to recommend creating separate config file instead of messing with main one.

### CODE REVIEW

File: users/autostart.rst [hunk 1]
```diff
@@ -198,16 +198,27 @@ On any distribution (Arch, Debian, Linux Mint, Ubuntu, openSUSE)
 Using Supervisord
 ~~~~~~~~~~~~~~~~~
 
-Add the following to your supervisor config file::
+Create the ``syncthing.conf`` file in supervisor's ``conf.d`` folder (the folder is located
+at ``/etc/supervisor/conf.d``)::
```
Comment (AudriusButkevicius) [inline users/autostart.rst]: In supervisors x folder which is actually /etc/foo/x, why not just /etc/foo/x straight away?

Comment (AudriusButkevicius) [reply users/autostart.rst]: Is it different between ubuntu/debian?

Comment (MarcinOrlowski) [reply users/autostart.rst]: Not sure I understand your question. Debian/Ubuntu installation uses `conf.d` subfolder as default location for custom configs which helps keeping all stuff organized.

Comment (AudriusButkevicius) [reply users/autostart.rst]: Sure, as long as they are in the same place no point talking about conf.d and then giving the full path, give the full path straight away.

Comment (MarcinOrlowski) [reply users/autostart.rst]: Done

**Pull request merged**